library/kani_macros/src/lib.rs

```diff
@@ -10,6 +10,9 @@
 
 // proc_macro::quote is nightly-only, so we'll cobble things together instead
 use proc_macro::TokenStream;
+use proc_macro2::TokenStream as TokenStream2;
```

Comment:
Two different proc macro crates?

Reply:
Both `syn` and `quote` use the `proc_macro2` crate. 

`proc_macro2` is a wrapper around `proc_macro`, except it allows you to use the defined types outside a procedural macro. 

I need `proc_macro2::TokenStream` because `quote::quote!(...)` expects inputs to be of that type.